src/main/flight/pid.c

```diff
@@ -351,6 +369,13 @@ void pidInitConfig(const pidProfile_t *pidProfile)
     itermRelax = pidProfile->iterm_relax;
     itermRelaxCutoffLow = pidProfile->iterm_relax_cutoff_low;
     itermRelaxCutoffHigh = pidProfile->iterm_relax_cutoff_high;
+
+#ifdef USE_ACRO_TRAINER
+    acroTrainerAngleLimit = pidProfile->acro_trainer_angle_limit;
+    acroTrainerLookaheadTime = (float)pidProfile->acro_trainer_lookahead_ms / 1000.0f;
```

Comment:
This explicit cast to float, as well as the one below, are not necessary. Let's not clutter the code with visual noise :)